packages/documentation/versioned_docs/version-1.50.0/for-developers/data-model.md

```diff
@@ -24,7 +24,7 @@ Currently, there is not a very clearly defined flow for modifying these document
 This includes:
 
 - [Blueprints](https://github.com/nrkno/sofie-core/blob/master/packages/corelib/src/dataModel/Blueprint.ts)
-- [Buckets](https://github.com/nrkno/sofie-core/blob/master/meteor/lib/collections/Buckets.ts)
+- [Buckets](https://github.com/nrkno/sofie-core/blob/master/packages/corelib/src/dataModel/Bucket.ts)
```

Comment:
We should probably let this be? This refers to historical information anyway, so will not align with the rest of documentation.